### ISSUE TITLE
There is a stray console.log() in production code

### ISSUE THREAD

**Issue by nicholaswmin** [motss/app-datepicker#35]
This piece of code `console.logs()`  for no apparent reason and seems to be a stray "leftover"

```
          // When datepicker has no button for date confirmation,
          // by asserting `autoUpdateDate` will trigger auto update on
          // date change.
          console.log(this.autoUpdateDate);
```

@ `app-datepicker.html - line 1029`

**Comment by motss** [comment]
@nicholaswmin Thanks for the issue. Mind submitting a PR to fix that?

**Comment by nicholaswmin** [comment]
@motss coming right up

**Comment by nicholaswmin** [comment]
@motss Is there a mismatch between what's at #master branch and what's published in bower or was this fixed?

**Comment by nicholaswmin** [comment]
Fixed in 8239a20bc388b1c4baf3e8dbd9ce16707c8222c2 - Closing
